FIDO2_messages_details.md

```diff
@@ -0,0 +1,188 @@
+responds with [FIDO2 Make Authenicator Data Response](FIDO2_messages_details.md#fido2-make-authenticator-data-response). See FIDO2 specification
+for definition of the values that are not explicitly defined here.
+
+**From AUX MCU:**  
+
+| byte 0 - 251 | byte 252 - 316    | byte 317 - 381 | byte 382 - 413        | byte 414 - 446   | byte 447 | byte 448 |
+|:-------------|:------------------|:---------------|:----------------------|------------------|:---------|:---------|
+| RPID         | User ID           | User name      | Display name          | Client data hash | Index    | New cred |
+
+**RPID**
+
+[RPID](https://www.w3.org/TR/webauthn/#rp-id) for the request.
+
+**User ID**
+
+[User ID](https://www.w3.org/TR/webauthn/#dom-publickeycredentialuserentity-id) associated witht the credential
+
+**Display name**
+
+[Display name](https://www.w3.org/TR/webauthn/#dom-publickeycredentialuserentity-displayname) as per FIDO2 spec.
+
+**Client data hash**
+
+[Client data hash](https://www.w3.org/TR/webauthn/#collectedclientdata-hash-of-the-serialized-client-data)
+
+**Index**
+
+Index of credential to get data for when we are iterating over all the credentials associated with an [RPID](https://www.w3.org/TR/webauthn/#rp-id). We start from 0 and then continue monotonically incrementing "Index" until MAIN MCU responds that no credential found for that index.
+
+**New cred**
+
+| Value  | Description                                                   |
+|:-------|:--------------------------------------------------------------|
+| 0x00   | We are getting an existing credential ([Get Assertion](https://fidoalliance.org/specs/fido-v2.0-ps-20190130/fido-client-to-authenticator-protocol-v2.0-ps-20190130.html#authenticatorGetAssertion))        |
+| 0x01   | We are creating a new credential (storing) ([Make Credential](https://fidoalliance.org/specs/fido-v2.0-ps-20190130/fido-client-to-authenticator-protocol-v2.0-ps-20190130.html#authenticatorMakeCredential))  |
+
+## FIDO2 Make Authenticator Data Response
+
+This message is sent as a response to request by the AUX MCU to make authenticator data for a FIDO2 credential.
+See FIDO2 specification for definition of the values that are not explicitly defined here.
+
+**From MAIN MCU:**
+
+| byte 0 - 31    | byte 32 - 63  | byte 64 - 67         | byte 68 - 71   | byte 72 - 103          |
+|:---------------|:--------------|:---------------------|:---------------|:-----------------------|
+| Credential ID  | RPID Hash     | Count (in BE format) | Flags          | Public key x component |
+
+| byte 104 - 135         | byte 136 - 199   | byte 200 - 215 | byte 216 - 219       | byte 220   |
+|:-----------------------|------------------|:---------------|:---------------------|:-----------|
+| Public key y component | Attest signature | AAGUID         | Credential ID length | Error code |
+
+**Credental ID**
+
+The [CredentialID](https://www.w3.org/TR/webauthn/#credential-id) for the entry to check
```

Comment:
created credential id no?

Reply:
Not sure what you mean.

Reply:
"for the entry to check" isn't correct, it would be "newly created credential id" no?